modules/node/handler/node_pagination_builder.e

```diff
@@ -0,0 +1,67 @@
+note
+	description: "Generic Node Pagination Builder Interface"
+	date: "$Date$"
+	revision: "$Revision$"
+
+deferred class
+	NODE_PAGINATION_BUILDER [G]
```

Comment:
I would name it

CMS_PAGINATION_BUILDER [G]

and the NODE_PAGE_BUILDER, name it CMS_NODE_PAGINATION_BUILDER